### ISSUE TITLE
Some of the line-heights in the text characteristic have been calculated inconsistently

### ISSUE THREAD

**Issue by joshwooding** [jpmorganchase/salt-ds#4224]
### Latest version

- [X] I have tested the latest version

### Description

![image](https://github.com/user-attachments/assets/6fbabda4-95f7-4cd1-8788-9f8d4fddbb1f)

The top table is current, the bottom in next.
The numbers marked in red are incorrect. lh = line height, e lh = expected line height. 

### Steps to reproduce

.

### Package name(s)

Theme (@salt-ds/theme)

### Browser

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Microsoft Edge

### Operating system

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Are you a JPMorgan Chase & Co. employee?

- [ ] I am an employee of JPMorgan Chase & Co.

**Comment by origami-z** [comment]
I can not recall any odd number line height occurs in memory, where in the process does this fall apart?

**Comment by joshwooding** [comment]
> I can not recall any odd number line height occurs in memory, where in the process does this fall apart?

We do have odd number line-heights (some of the display text), but I've mentioned it in #4226 that you get a worse outcome. #4223 is caused by this.

**Comment by bhoppers2008** [comment]
@joshwooding @dplsek @pseys is this still relevant?

**Comment by joshwooding** [comment]
@bhoppers2008 Yes and no... There is one issue with h1 in td that's inconsistent between code and figma. There are other "issues" but fixing them would potentially be a breaking change, so would have to discuss pros and cons.

**Comment by mark-tate** [comment]
Q3 goal to workout what we will do, make changes in pixels or fail-forward to rems.
Q4+ goal to look at rem

**Comment by joshwooding** [comment]
Closing as the aim will be to fix this in a new brand-aligned theme, we are unlikely to change this in the legacy/current "next" theme.
